src/modules/gtk2/producer_pango.c

```diff
@@ -413,8 +423,8 @@ static void refresh_image( mlt_frame frame, int width, int height )
 	int weight = mlt_properties_get_int( producer_props, "weight" );
 	int rotate = mlt_properties_get_int( producer_props, "rotate" );
 	int size = mlt_properties_get_int( producer_props, "size" );
-	int width_crop = mlt_properties_get_int( producer_props, "width_crop" );
-	int width_fit = mlt_properties_get_int( producer_props, "width_fit" );
+	int width_type = mlt_properties_get_int( producer_props, "width_type" );
+	int width_value = mlt_properties_get_int( producer_props, "width_value" );
```

Comment:
You are changing existing behavior, which is not permitted (unless it is an important bug fix). People has written apps, scripts, and compositions (e.g. MLT XML) that may be using these properties (width_crop, width_fit). If width_crop and width_fit were unreleased, I would say OK; it was still in development. However, it was released with v0.6.2.